docs/docs/installation/architecture.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Architecture
+hide_title: true
+sidebar_position: 1
+version: 1
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Architecture
+
+This page is meant to give new administrators an understanding of Superset's components.
+
+## Components
+
+A Superset installation is made up of these components:
+1. The Superset application itself
```

Comment:
Should we call it "the Superset [python] backend"?

Reply:
Should we call out "The Superset frontend" - a collection of React application bundled with webpack and using antd, ...

Reply:
I mention this in a suggestion below in the description of the Superset application.